### DISCUSSION TITLE
Request to Join InstructLab Org for IBM Consulting's Red Hat Hackathon Contribution

### DISCUSSION THREAD

**Discussion by artreimus** [instructlab/community#359] [General]
Hi InstructLab Team,

My name is Arthur, and I’m an AI Engineer at IBM. My team and I are participating in IBM Consulting's Red Hat Hackathon, and our project focuses on contributing new features to the InstructLab GUI. However, when attempting to run the GUI on my local machine, I encountered a 404 error with the message: "Whoops! You need to be a member of the InstructLab org to access this site. Try joining and then come back!"

Could you please assist me in joining the InstructLab organization so that my team and I can move forward with our contributions? 

Thank you!

**Comment by jjasghar** [comment]
Hi! I'm an IBMer, please ping me on slack (@awesome) and I can get you the link.
